src/org/labkey/test/TestFileUtils.java

```diff
@@ -212,7 +212,7 @@ public static File getSampleData(String relativePath)
             File checkFile = new File(sampledataDir, relativePath);
             if (checkFile.exists())
             {
-                if (foundFile != null && !foundFile.equals(checkFile))
+                if (foundFile != null && foundFile.length() != checkFile.length()) // Allow duplicate files to ease migration
```

Comment:
Would a more appropriate check be to compare the hash of the file?
    if(foundFile != null && foundFile.hash() != checkFile.hash()))

Reply:
`File` doesn't have any built in hashing functionality.
This isn't meant to be a rock-solid verification that they're the same file. Them probably being the same file is sufficient. We'll go back to being strict once we've gotten all of the sample data out of SVN.